pkg/syncer/storagepool/service.go

```diff
@@ -193,13 +193,35 @@ func ResetVC(ctx context.Context, vc *cnsvsphere.VirtualCenter) {
 		log.Errorf("Failed to connect to SPBM service. Err: %+v", err)
 		return
 	}
-	log.Infof("Resetting VC connection in StoragePool service")
+	log.Info("Resetting VC connection in StoragePool service")
 	defaultStoragePoolServiceLock.Lock()
 	defer defaultStoragePoolServiceLock.Unlock()
-
-	defaultStoragePoolService.spController.vc = vc
-	defaultStoragePoolService.scWatchCntlr.vc = vc
-	defaultStoragePoolService.migrationCntlr.vc = vc
-	// PC listener will automatically reestablish its session with VC.
-	log.Debugf("Successfully reset VC connection in StoragePool service")
+	resetspControllerVC := false
+	resetscwatchControllerVC := false
+	resetmigrationCntlrVC := false
```

Comment:
nit: Can rename to `resetmigrationControllerVC` to keep it consistent

Reply:
done.